lib/bootloader_setup.pm

```diff
@@ -1246,7 +1246,7 @@ Returns the array of the boot parameters.
 =cut
 
 sub parse_bootparams_in_serial {
-    my $parsed_string = wait_serial(qr/command line:.*/msi);
+    my $parsed_string = wait_serial(qr/command line:.*[[]/msi);
```

Comment:
Hmm, I'm confused why do we need to wrap `[` in square braces? As we have single character, I believe that we can add just `[`, which we just need to escape.

Reply:
It's a special character so it will work either with the above syntax or with "\[" (is that what you mean "escape"?) . I just thought the first way is a bit more readable . WDYT?

Reply:
Usually `[]` is used for the range of characters, like `[a-z]` or `[abc]`. As in this case we have just `[`, I would suggest putting just `\[`.

Reply:
Done